### ISSUE TITLE
global nightwatch needs global cucumber

### ISSUE THREAD

**Issue by tmpst** [mucsi96/nightwatch-cucumber#102]
running `$ nightwatch` from the command line fails if cucumber is not also installed globally.

The fix is to install cucumber globally `$ npm i -g cucumber` if you want to use a global install of nightwatch, and the documentation needs to be updated to reflect this.

**Comment by mucsi96** [comment]
Thanks for your feedback! I tried it with globally installed Nightwatch. But I got the following error `Cannot find module 'nightwatch-cucumber`. That means Nightwatch.js is not handling JS based configuration files correctly if installed globally.  As I can't fix this in the plugin I will remove the section about global Nightwatch from the docs for now. I will raise an issue for Nightwatch and close this for now.
